tests/openssl-rand-provider/openssl_rng_tester.c

```diff
@@ -0,0 +1,103 @@
+#include <openssl/provider.h>
+#include <openssl/rand.h>
+#include <openssl/rsa.h>
+#include <openssl/evp.h>
+#include <openssl/params.h>
+#include <openssl/core_names.h>
+#include <assert.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include "env.h"
+
+static void test_random() {
+    unsigned char bytes[300];
+    int ret = RAND_bytes(bytes, 300);
+    fprintf(stderr, "%i\n", ret);
```

Comment:
nit: coding style: I very much like KNF which asks for a blank line between variable declarations and code.